docs/sources/release-notes/v2.3.md

```diff
@@ -29,6 +29,11 @@ For the complete list of changes, see the [Changelog](https://github.com/grafana
 - **Added `backfill` command to mimirtool to upload Prometheus TSDB blocks:**
   The `mimirtool backfill` command will make the process of back-filling Prometheus TSDB blocks, using an API available in the compactor, even easier which allows for a migration to Grafana Mimir while keeping your previous Prometheus TSDB data.
 
+- **Helm Helm Chart improvements:** The Mimir Helm chart is the best way to install Mimir on Kubernetes. As part of the Mimir 2.3 release, we’re also releasing version 3.1 of the Helm chart. Notable enhancements follow. For the full list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+  - The MinIO sub chart dependency is upgraded from a deprecated Helm chart (https://helm.min.io/) to the supported chart's version (https://charts.min.io/). This includes a **breaking change** of how the administrator password is set, however as the _built-in_ MinIO is not recommended for production, this change did not warrant a new major version of the Mimir Helm chart.
+  - The backfill API endpoints are now exposed on the Nginx gateway.
+  - Default to injecting the `no_auth_tenant` from the Mimir configuration as the value for `X-Scope-OrgID` in Nginx instead of hard coded `anonymous`.
```

Comment:
not feeling strongly about this
```suggestion
  - Default to injecting the `no_auth_tenant` from the Mimir configuration as the value for `X-Scope-OrgID` in Nginx instead of hard coded `anonymous`.
  - Memberlist now uses DNS service-discovery by default. This allows Mimir only join a subset of the memberlist cluster when starting. This makes startup in large clusters faster.
```

Reply:
yeah, I felt like that's more an implementation detail

Reply:
- Memberlist now uses DNS service-discovery by default, which should decrease startup time for large Mimir clusters.
^I tried rewriting to remove some of the implementation detail and emphasize impact. 

I'm not sure how if at all I should change my suggestion based on @pracucci 's review comment though.

Reply:
I also updated the language for the nginx callout to also try to call out the rationale for the change. 

  - Nginx now sets the value of the `X-Scope-OrgID` header equal to the value of Mimir's `no_auth_tenant` flag by default. The previous release had set `X-Scope-OrgID` to the hard coded value of `anonymous` by default which [complicated the process of migrating to Mimir](https://github.com/grafana/mimir/issues/2563).

Reply:
committing this but changing "flag" to "parameter" since we only call CLI flags "flags", the YAML contains parameters